backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,100 @@
+create table project
+(
+    project_id  bigserial     not null primary key,
+    name        varchar(256)  not null unique,
+    description varchar(1000) null
+);
+
+create table test
+(
+    test_id bigserial    not null primary key,
+    name    varchar(256) not null unique
+);
+
+create table subtest
+(
+    subtest_id      bigserial      not null primary key,
+    test_id         bigint         not null,
+    name            varchar(256)   not null unique,
```

Comment:
I am sorry, must likely that's a stupid question, but since we know that `name` is unique, why do we need `bigserial` ID, can't we use `name` as the primary key? I mean not only here, but also in `test` and `project`.

Reply:
https://stackoverflow.com/questions/15477005/what-are-the-pros-and-cons-for-choosing-a-character-varying-data-type-for-primar

It's a classical debate. We can go any way we prefer. By default I prefer approach with surrogate primary key.